### PR TITLE
Removing unused variable warnings

### PR DESCRIPTION
**Pull request by ulysse314**
Hello,

This commit reduces the number of unused variable warnings.

Thanks,

### CODE REVIEW

File: INA219.cpp [hunk 1]
```diff
@@ -77,16 +77,11 @@ void INA219::begin() {
 
 void INA219::calibrate(float shunt_val, float v_shunt_max, float v_bus_max, float i_max_expected) {
     uint16_t digits;
-    float i_max_possible, min_lsb, max_lsb, swap;
+    float min_lsb, swap;
 
-#if (INA219_DEBUG == 1)
```
Comment (flav1972) [inline INA219.cpp]: Could you put all the variables definitions here ? It looks more like classic C style.

**Review by flav1972 — CHANGES_REQUESTED** [review]
can you please put the declaration of variables block at the beginning of the method ?

**Comment by ulysse314** [comment]
Done

**Comment by ulysse314** [comment]
I'm not convinced about moving the variables at the beginning of the method since it is c++, and not c. Also I feel it is better to have the variables next to the code, since it seems easier to read with one debug block.

**Review by flav1972 — APPROVED** [review]

**Pull request merged**